skaffold.yaml

```diff
@@ -33,6 +36,8 @@ deploy:
             uid: '{{ env "CLUSTER_NAME" | default "cluster-uid-123456789" }}'
           otel:
             endpoint: timeseries-mock-service:9082
+            swi_endpoint_check: 
+              enabled: false
```

Comment:
Why?

Reply:
In other words, the startup of `timeseries-mock-service` now takes much longer and the check times out because of it. OK.